t/YachtSpecs.t

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+use Test::More;
+use DDG::Test::Spice;
+
+spice is_cached => 1;
+
+ddg_spice_test(
+    [qw( DDG::Spice::YachtSpecs)],
+    'yacht maluhea' => test_spice(
+        '/js/spice/yacht_specs/maluhea',
+        call_type => 'include',
+        caller => 'DDG::Spice::YachtSpecs'
+    ),
+    'custom line yacht maluhea' => undef,
+'private yacht maluhea' => undef
```

Comment:
@elephantRunAway Do you mind adding some `undef` trigger tests here for two of the skip words added in  5be5021 :+1: